src/components/Instructions.component.js

```diff
@@ -5,16 +5,25 @@ import './../styles/instructions.css'
 
 class Instructions extends Component {
   render() {
-    return (
-      <div className="instructions">
-        Follow the instructions on the README to get started!
-        <img
-          className="instructions__logo"
-          src="https://uiuc.hack4impact.org/img/colored-logo.png"
-          alt="h4i logo"
-        />
-      </div>
-    )
+    let shouldDisplayImage = this.props.shouldDisplayImage
```

Comment:
use `const`, or just skip storing it as a variable, and on the next line, just do
```javascript
if (this.props.shouldDisplayImage) {
```